frontend/src/helpers/connectionHelper.ts

```diff
@@ -49,6 +51,13 @@ export function clearConnectionError(connection: IConnection) {
   emit('connection', connection)
 }
 
+export function getConnectionIds(state: ApplicationState) {
+  const { device, connections } = state.backend
+  let ids = connections.map(c => c.id)
```

Comment:
why do you need a `let`?

Reply:
well I'm pushing to the array in the next line, so I assume it's gotta be let or var... not const if it's changing.  🤷‍♂️